nemo/collections/common/parts/adapter_modules.py

```diff
@@ -146,3 +158,67 @@ class LinearAdapterConfig:
     dropout: float = 0.0
     adapter_strategy: Optional[dict] = adapter_mixin_strategies.ResidualAddAdapterStrategyConfig()
     _target_: str = "{0}.{1}".format(LinearAdapter.__module__, LinearAdapter.__name__)
+
+
+class ParallelLinearAdapter(AbstractAdapterModule):
```

Comment:
I think it would be better to move this class into somewhere inside the NLP collection - that way no circular import and nk need for apex checks in common dir.